etl/steps/data/garden/hmd/2024-11-19/hfd.meta.yml

```diff
@@ -0,0 +1,288 @@
+
+      sdmab:
+        title: |-
+          <% set title = "Standard deviation in period mean ages at birth" %>
+          {definitions.others.title}
+        description_short: |-
+          <% if birth_order == 'total' %>
+          How much variation there is in the ages of mothers when they give birth in a specific year.
+          <%- else %>
+          How much variation there is in the ages of mothers when they have their{definitions.others.bo_1} child in a specific year.
+          <%- endif %>
+        unit: "years"
+        description_key:
+          - Measures the spread of ages at which women give birth.
+          - A low standard deviation indicates that most births occur around the same age, while a high standard deviation suggests a wider range of ages.
+          - Useful for understanding the diversity of reproductive timing.
+
+      sdmab40:
+        title: |-
+          <% set title = "Standard deviation in period mean ages at birth by age 40" %>
+          {definitions.others.title}
+        description_short: |-
+          <% if birth_order == 'total' %>
+          How much variation there is in the ages of mothers under 40 when they give birth.
+          <%- else %>
+          How much variation there is in the ages of mothers under 40 when they have their{definitions.others.bo_1} child in a specific year.
+          <%- endif %>
+        unit: "years"
+        description_key:
+          - Reflects variability in the timing of births up to age 40.
+          - Helps to understand how concentrated or spread out early childbearing is within the population.
+
+      tmab:
+        title: |-
+          <% set title = "Period table mean ages at birth" %>
+          {definitions.others.title}
+        description_short: |-
+          <% if birth_order == 'total' %>
+          The average age of mothers at birth, considering the number of children they already have.
+          <%- else %>
+          The average age of mothers at the birth of their{definitions.others.bo_1} child.
+          <%- endif %>
+        unit: "years"
+        description_key:
+          - Derived from fertility tables that account for both age and parity.
+          - Provides a more detailed understanding of the timing of births across different birth orders.
+
+      # Births and birth rate
+      cbr:
+        title: |-
+          <% set title = "Crude birth rate" %>
+          {definitions.others.title}
+        description_short: |-
+          The number of{definitions.others.bo_1} live births per 1,000 people in a given year. It is not standardized for the age structure of the population.
+        unit: "births per 1,000 people"
+        description_key:
+          - The rate is calculated by dividing the total number of{definitions.others.bo_1} live births occurring in a given year by person-years lived by all population in that year.
+          - Simple measure of birth intensity, not adjusted for age or parity.
```

Comment:
It's not clear what parity ies, and maybe it can be combined with the next bullet point

Reply:
totally. I'll discuss with Saloni adding a DoD for this term.